Kudu.Services/Diagnostics/DiagnosticsController.cs

```diff
@@ -79,53 +85,53 @@ public HttpResponseMessage GetRecentLogs(int top = 100)
 
                 var vfsBaseAddress = UriHelper.MakeRelative(UriHelper.GetBaseUri(request), "api/vfs");
 
                 // Open files in order to refresh (not update) the timestamp and file size.
                 // This is needed on Linux due to the way that metadata for files on the CIFS
                 // mount gets cached and not always refreshed. Limit to 10 as a safety.
-
-                foreach (var filename in currentDockerLogFilenames.Take(10))
+                if (!OSDetector.IsOnWindows())
                 {
-                    using (var file = File.OpenRead(filename))
+                    foreach (var filename in currentContainerLogFilenames.Take(10))
                     {
-                        // This space intentionally left blank
+                        using (var file = File.OpenRead(filename))
+                        {
+                            // This space intentionally left blank
+                        }
                     }
                 }
 
-                var responseContent = currentDockerLogFilenames.Select(p => CurrentDockerLogFilenameToJson(p, vfsBaseAddress.ToString()));
+                var responseContent = currentContainerLogFilenames.Select(p => CurrentContainerLogFilenameToJson(p, vfsBaseAddress.ToString()));
 
                 return Request.CreateResponse(HttpStatusCode.OK, responseContent);
             }
         }
 
-        // Route only exists for this on Linux
-        // Grabs "currently relevant" Docker logs from the LogFiles folder
+        // Route exists for this on Linux and Windows
+        // Grabs "currently relevant" Container logs from the LogFiles folder
         // and returns them in a zip archive
         [HttpGet]
         [SuppressMessage("Microsoft.Usage", "CA2202", Justification = "The ZipArchive is instantiated in a way that the stream is not closed on dispose")]
-        public HttpResponseMessage GetDockerLogsZip()
+        public HttpResponseMessage GetContainerLogsZip()
         {
-            using (_tracer.Step("DiagnosticsController.GetDockerLogsZip"))
+            using (_tracer.Step("DiagnosticsController.GetContainerLogsZip"))
             {
-                // Also search for "today's" files in sub folders. Windows containers archives log files
-                // when they reach a certain size.
-                var currentDockerLogFilenames = GetCurrentDockerLogFilenames(SearchOption.AllDirectories);
+                var currentContainerLogFilenames = GetCurrentContainerLogFilenames(SearchOption.TopDirectoryOnly);
 
                 HttpResponseMessage response = Request.CreateResponse();
-                response.Content = ZipStreamContent.Create(String.Format("dockerlogs-{0:MM-dd-HH-mm-ss}.zip", DateTime.UtcNow), _tracer, zip =>
+                response.Content = ZipStreamContent.Create(String.Format("containerlogs-{0:MM-dd-HH-mm-ss}.zip", DateTime.UtcNow), _tracer, zip =>
```

Comment:
If we change the name of the .zip file we may break customers (for example: if they have automation to download the logs periodically)

Reply:
yes I thought of it, but we dont have any better option..We dont want to keep it as docker logs for sure. Open for ideas...